<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1001</width>
      <height>651</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Загрузить модель</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>398</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="2" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Центр масштаб.:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Углы поворота (градусы):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="dspRotY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RotX:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label001">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Перемещение:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="QPushButton" name="btnRotate">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="dspRotZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Масштабирование:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="dspScaleZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QDoubleSpinBox" name="dspRotYcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="dspMoveY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ScaleZ:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="4">
      <widget class="QPushButton" name="btnScale">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="dspScaleX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QDoubleSpinBox" name="dspScaleXcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QDoubleSpinBox" name="dspScaleYcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ScaleX:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QDoubleSpinBox" name="dspScaleZcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RotZ:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="dspRotX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ScaleY:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBoxAutoRotation">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Авто поворот</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QDoubleSpinBox" name="dspRotZcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RotY:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="dspScaleY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPushButton" name="btnMove">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Переместить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QDoubleSpinBox" name="dspRotXcenter">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Центр поворота:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="dspMoveZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="dspMoveX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>590</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Сохранить модель</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAutoRotation">
   <property name="text">
    <string>AutoRotation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
